Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -66,7 +66,20 @@ kubectl -n rook-ceph get pod -l app=rook-ceph-rgw
 
 ### Connect to an External Object Store
 
-Rook can connect to existing RGW gateways to work in conjunction with the external mode of the `CephCluster` CRD.
+Rook can connect to existing RGW gateways to work in conjunction with the external mode of the `CephCluster` CRD. First you need to create a `rgw-admin-ops-user` user in the Ceph cluster with the following command assigning following caps:
```

Comment:
We are updating rook docs to avoid terms like "you need to", or "you should". The docs should be direct.
```suggestion
Rook can connect to existing RGW gateways to work in conjunction with the external mode of the `CephCluster` CRD. First, create a `rgw-admin-ops-user` user in the Ceph cluster with the following command assigning following caps:
```